src/components/transport_manager/test/transport_manager_impl_test.cc

```diff
@@ -49,235 +54,276 @@ using ::testing::Return;
 
-  tm.TestHandle(test_event);
-}
+    TransportAdapterEvent test_event(type,
+                                     mock_adapter_,
+                                     mac_address_,
+                                     application_id_,
+                                     test_message_,
+                                     error_);
 
-void TransportManagerImplTest::HandleSearchFail() {
-  int type = static_cast<int>(
-      TransportAdapterListenerImpl::EventTypeEnum::ON_SEARCH_FAIL);
+    EXPECT_CALL(*tm_listener_, OnFindNewApplicationsRequest());
 
-  TransportAdapterEvent test_event(type, mock_adapter, mac_address_,
-                                   application_id, test_message_, error_);
+    tm_.TestHandle(test_event);
+  }
 
-  EXPECT_CALL(*tm_listener, OnScanDevicesFailed(_));
+  void HandleConnectionClosed() {
+    const int type = static_cast<int>(
+        TransportAdapterListenerImpl::EventTypeEnum::ON_DISCONNECT_DONE);
 
-  tm.TestHandle(test_event);
-}
+    TransportAdapterEvent test_event(type,
+                                     mock_adapter_,
+                                     mac_address_,
+                                     application_id_,
+                                     test_message_,
+                                     error_);
 
-void TransportManagerImplTest::HandleFindNewApplicationsRequest() {
-  int type = static_cast<int>(TransportAdapterListenerImpl::EventTypeEnum::
-                                  ON_FIND_NEW_APPLICATIONS_REQUEST);
+    EXPECT_CALL(*tm_listener_, OnConnectionClosed(application_id_));
 
-  TransportAdapterEvent test_event(type, mock_adapter, mac_address_,
-                                   application_id, test_message_, error_);
+    tm_.TestHandle(test_event);
+  }
 
-  EXPECT_CALL(*tm_listener, OnFindNewApplicationsRequest());
+  void HandleDisconnectionFailed() {
+    const int type = static_cast<int>(
+        TransportAdapterListenerImpl::EventTypeEnum::ON_DISCONNECT_FAIL);
 
-  tm.TestHandle(test_event);
-}
+    TransportAdapterEvent test_event(type,
+                                     mock_adapter_,
+                                     mac_address_,
+                                     application_id_,
+                                     test_message_,
+                                     error_);
 
-void TransportManagerImplTest::HandleConnectionClosed() {
-  int type = static_cast<int>(
-      TransportAdapterListenerImpl::EventTypeEnum::ON_DISCONNECT_DONE);
+    EXPECT_CALL(*tm_listener_, OnDisconnectFailed(device_handle_, _));
 
-  TransportAdapterEvent test_event(type, mock_adapter, mac_address_,
-                                   application_id, test_message_, error_);
+    tm_.TestHandle(test_event);
+  }
 
-  EXPECT_CALL(*tm_listener, OnConnectionClosed(application_id));
+  void UninitializeTM() {
+    EXPECT_CALL(*mock_adapter_, Terminate());
+    ASSERT_EQ(E_SUCCESS, tm_.Stop());
+  }
 
-  tm.TestHandle(test_event);
-}
+  MockTransportManagerImpl tm_;
```

Comment:
@VVeremjova I suggest to use more clear name (like `mock_transport_manager_`) as you did in line below.